.github/workflows/maven.yml

```diff
@@ -22,32 +22,11 @@ jobs:
       with:
         path: main
```

Comment:
Do you really want to keep the `main` directory here?

Previously, the code was checked out in the current directory (see [this commit](https://github.com/powsybl/powsybl-entsoe/pull/190/commits/1cb27c7b8ed2b60886346cdc7857e7d06a9ed407), which introduced the workflow changes)